doc/esmvalcore/datafinder.rst

```diff
@@ -1,7 +1,251 @@
+  [variable_short_name]/[grid]``;
+* CMIP6 path for ETHZ: ``ROOT-ETHZ/[experiment]/[mip]/[variable_short_name]/[dataset_name]/[ensemble]/[grid]``
+
+From the ESMValTool user perspective the number of data input parameters is
+optimized to allow for ease of use. We detail this procedure in the next
+section.
+
+.. _data-retrieval:
+
+Data retrieval
+==============
+Data retrieval in ESMValTool has two main aspects from the user's point of
+view: 
+
+* data can be found by the tool, subject to availability on disk;
+* it is the user's responsibility to set the correct data retrieval parameters;
+
+The first point is self-explanatory: if the user runs the tool on a machine
+that has access to a data repository or multiple data repositories, then
+ESMValTool will look for and find the avaialble data requested by the user.
+
+The second point underlines the fact that the user has full control over what
+type and the amount of data is needed for the analyses. Setting the data
+retrieval parameters is explained below.
+
+Setting the correct root paths
+------------------------------
+The first step towards providing ESMValTool the correct set of parameters for
+data retrieval is setting the root paths to the data. This is done in the user
+configuration file ``config-user.yml``. The two sections where the user will
+set the paths are ``rootpath`` and ``drs``. ``rootpath`` contains pointers to
+``CMIP``, ``OBS``, ``default`` and ``RAWOBS`` root paths; ``drs`` sets the type
+of directory structure the root paths are structured by. It is important to
+first discuss the ``drs`` parameter: as we've seen in the previous section, the
+DRS as a standard is used for both file naming conventions and for directory
+structures. 
+
+.. _config-user-drs:
+
+Explaining ``config-user/drs: CMIP5:`` or ``config-user/drs: CMIP6:``
+---------------------------------------------------------------------
+Whreas ESMValTool will **always** use the CMOR standard for file naming (please
+refer above), by setting the ``drs`` parameter the user tells the tool what
+type of root paths they need the data from, e.g.: 
+
+  .. code-block:: yaml
+
+   drs:
+     CMIP6: BADC
+
+will tell the tool that the user needs data from a repository structured
+according to the BADC DRS structure, i.e.:
+
+``ROOT/[institute]/[dataset_name]/[experiment]/[ensemble]/[mip]/[variable_short_name]/[grid]``;
+
+setting the ``ROOT`` parameter is explained below. This is a
+strictly-structured repository tree and if there are any sort of irregularities
+(e.g. there is no ``[mip]`` directory) the data will not be found! ``BADC`` can
+be replaced with ``DKRZ`` or ``ETHZ`` depending on the existing ``ROOT``
+directory structure.
+The snippet
+
+  .. code-block:: yaml
+
+   drs:
+     CMIP6: default
+
+is another way to retrieve data from a ``ROOT`` directory that has no DRS-like
+structure; ``default`` indicates that the data lies in a directory that
+contains all the files without any structire.
```

Comment:
Should be "structure" instead of "structire"?

Reply:
I guess so.